modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -1208,6 +1212,7 @@ added-all=Added All
 added-any=Added Any
 added-via-api=Added via API
 adding-a-stepper-fragment-inside-a-simple-form-will-turn-it-into-a-multistep-form=Adding a stepper fragment inside a simple form will turn it into a multistep form. Are you sure you want to continue?
+adding-categories-and-tags-will-preserve-the-ones-already-applied-to-the-currently-selected-assets=Adding categories and tags will preserve the ones already applied to the currently selected items.
```

Comment:
probably is better to have the key
`adding-categories-and-tags-will-preserve-the-ones-already-applied`